docker/docker-compose.yml

```diff
@@ -63,8 +63,12 @@ services:
     depends_on:
       - db
       - maildev
-    build: .
-    image: jetpack_wordpress:localbuild
+    build:
+      context: .
+      args:
+        PHP_VERSION: ${DOCKER_PHP_VERSION}
```

Comment:
What do you think if we drop `DOCKER_` out from this? 

I'd love to be able to type `PHP_VERSION=5 yarn docker:up` where `5` could be any version we didn't include in yarn scripts.

Reply:
Noting here also that [variables modifying Docker/Docker-compose CLI](https://docs.docker.com/compose/reference/envvars/) all start with `DOCKER_` so we shouldn't create names that can be confused with those.